panel/src/panel/content.js

```diff
@@ -110,22 +110,32 @@ export default (panel) => {
 		 * Saves any changes
 		 */
 		async save() {
-			if (this.isProcessing === true) {
-				return true;
-			}
```

Comment:
If we simply ignore requests while `isProcessing`, it could be that that last save request at the end of input is ignored if the backend is really slow (as previous requests won't have completed). We need to remove this to ensure that the latest request gets processed.